src/Connection.php

```diff
@@ -141,6 +142,18 @@ public function count()
     }
 
     /**
+     * Check if the connection is still active.
+     *
+     * @throws InvalidResourceException If connection was closed
+     *
+     * @return bool
+     */
+    public function isOpen(): bool
```

Comment:
This library is a wrapper for the imap extension, I would prefer to stick to the extension names: what about renaming this API to `ping`?